vllm/model_executor/models/llama.py

```diff
@@ -428,7 +434,9 @@ def load_weights(self, weights: Iterable[Tuple[str,
                 param = params_dict[scale_name]
                 weight_loader = getattr(param, "weight_loader",
                                         default_weight_loader)
-                loaded_weight = loaded_weight[0]
+                if loaded_weight.shape:
```

Comment:
Same for mllama.py?

Reply:
I targeted only the models which unconditionally do this logic